x/mongo/driver/operation/ismaster.go

```diff
@@ -201,6 +208,11 @@ func (im *IsMaster) command(dst []byte, _ description.SelectedServer) ([]byte, e
 	if im.maxAwaitTimeMS != nil {
 		dst = bsoncore.AppendInt64Element(dst, "maxAwaitTimeMS", *im.maxAwaitTimeMS)
 	}
+	if im.loadBalanced {
+		// The loadBalanced parameter should only be added if it's true. We should never explicitly send
+		// loadBalanced=false per the load balancing spec.
```

Comment:
I can't find this statement in the spec. It does say that `loadBalanced=false` and `loadBalanced` unset should have the same behavior, though.

Reply:
Under the "Monitoring" section, the design says

> If the loadBalanced=true connection string option is not specified, the driver MUST omit the option in connection handshakes.